lib/SILOptimizer/Mandatory/SendNonSendable.cpp

```diff
@@ -2475,15 +2475,13 @@ static void addSendableFixIt(const NominalTypeDecl *nominal,
 /// Add Fix-It text for the given generic param declaration type to adopt
 /// Sendable.
 static void addSendableFixIt(const GenericTypeParamDecl *genericArgument,
-                             InFlightDiagnostic &diag, bool unchecked) {
+                             InFlightDiagnostic &diag) {
```

Comment:
@gottesmm This is just copy and pasted from TypeCheckConcurrency.cpp. Perhaps it should be removed to lib/AST/...